policystorage/nomad.go

```diff
@@ -98,6 +98,7 @@ func validate(policy *api.ScalingPolicy) []error {
 	strategyList, ok := policy.Policy["strategy"].([]interface{})
```

Comment:
question: why is this `[]interface{}` (i.e., []map[string]interface{} below)? why does this work?

Reply:
@jrasell , @lgfa29 : what about this? ^

Reply:
i'm guessing it's some weird Go thing I didn't know about?

Reply:
I am honestly not sure. Unless we plan/planned on having scaling policies support a list of strategies, I believe we could just perform the `map[string]interface{}` cast. This is all conjecture without testing though.

Reply:
`hcl.DecodeObject` [here](https://github.com/hashicorp/nomad/blob/master/jobspec/parse_group.go#L350) turns `strategy` into a list.